README.md

```diff
@@ -97,7 +97,7 @@ Each Puppet module contains the following ```.pp``` files.
 
 ## Customizations
 
-1.  When updating the `<puppet_environment>/modules/ob_common/manifests/params.pp` of **Puppetmaster** to point the created databases, 
+1.  When updating the `<puppet_environment>/modules/ob_common/manifests/params.pp` file of **Puppetmaster** to point the created databases, 
```

Comment:
We can get rid of the numbering for this section

Reply:
Done